pkg/apiserver/common/common.go

```diff
@@ -146,34 +148,102 @@ func (s *Service) getKinds(c *gin.Context) {
 	c.JSON(http.StatusOK, kinds)
 }
 
-// @Summary Get the labels of the specified pod from Kubernetes cluster.
-// @Description Get the labels of the specified pod from Kubernetes cluster.
+// @Summary Get the labels of the pods in the specified namespace from Kubernetes cluster.
+// @Description Get the labels of the pods in the specified namespace from Kubernetes cluster.
 // @Tags common
 // @Produce json
-// @Success 200 {array} Label
+// @Success 200 {array} map[string][]string
```

Comment:
`ParseComment error in file pkg/apiserver/common/common.go :can not find schema type: "common.map"`